src/main/java/core/selenium/WebDriverConfig.java

```diff
@@ -12,6 +12,8 @@
 
 package core.selenium;
 
+import core.util.Log;
+
 import java.io.FileInputStream;
 import java.io.InputStream;
 import java.util.Properties;
```

Comment:
write again your documentation at the bottom, must be written in third person

Reply:
All in my documentations is in third person.